WordCount/WordCountAPI/Data/Models/CompositeKeyPair.cs

```diff
@@ -21,23 +25,34 @@ public CompositeKeyPair([NotNull]T1 first, [NotNull]T2 second)
 
         public override bool Equals(object obj)
         {
-            if (ReferenceEquals(null, obj)) return false;
-            if (ReferenceEquals(this, obj)) return true;
-            if (obj.GetType() != this.GetType()) return false;
-            return Equals((CompositeKeyPair<T1, T2>) obj);
+            if (ReferenceEquals(null, obj))
+            {
+                return false;
+            }
+
+            if (ReferenceEquals(this, obj))
+            {
+                return true;
+            }
+
+            if (obj.GetType() != this.GetType())
+            {
+                return false;
+            }
+
+            return Equals((CompositeKeyPair<T1, T2>)obj);
         }
 
         public override int GetHashCode()
         {
-            unchecked // Overflow is fine, just wrap
+            // Overflow is fine, just wrap
+            unchecked
             {
                 int hash = 851153;
-                hash = hash * 228601 + First.GetHashCode();
-                hash = hash * 826151 + Second.GetHashCode();
+                hash = (hash * 228601) + First.GetHashCode();
```

Comment:
Why are brackets needed here when multiplication is done before addition?

Reply:
it is done to be explicit